messages/data.export.bulk.md

```diff
@@ -1,37 +1,30 @@
 # summary
 
-Bulk export records from an org.
+Bulk export records from an org into a file using a SOQL query. Uses Bulk API 2.0.
 
 # description
 
-Specify a SOQL query to export fields of an object and get them in CSV or JSON format.
+You can use this command to export millions of records from an org, either to migrate data or to back it up.
 
-Use this command to export thousands of records from an org, either to migrate data or backup.
+Use a SOQL query to specify the fields of a standard or custom object that you want to export. Specify the SOQL query either at the command line with the --query flag or read it from a file with the --query-file flag; you can't specify both flags. You must write the records to a file, in either CSV or JSON format.
```

Comment:
> You must write the records to a file ...

This sounds like the user has to manually do it, maybe mention that `--output-file` is required and is where records will be saved?

Reply:
Good point!  Let me update it.